mira2_plugin_sparcomulti.i

```diff
@@ -76,32 +77,34 @@ func parse_options(plugin, opt)
   h_set, opt, an_option=my_needed_value;
 
   /* Read SPARCO settings */
-  sparco = opt.sparco_model;
+  sparco = strupper(opt.sparco_model);
```

Comment:
FITS keyword must be case insensitive, so writing/reading fits keyword mess up your case sensitive keyword. As a simple solution I change every keyword to uppercase